text.asm

```diff
@@ -69,21 +69,21 @@ _BoulderText::
 
 _MartSignText::
         ;     12345678901234567"
-	text "lo do dakli cu"    ;text "All your item"
-	line "culno vi"          ;line "needs fulfilled!"
-        cont "lo pokma zarci"    ;cont "#MON MART"
+	text "mi'a vecnu ro da"    ;text "All your item"
+	line "poi do nitcu ke'a!"          ;line "needs fulfilled!"
+    cont "i POKMONE ZARCI"    ;cont "#MON MART"
```

Comment:
Your editor is mixing tabs and spaces. The original uses tabs and we should probably stick to them.

(I just realized my editor has been making a mess of this too; I'll fix that on my end too.)